### ISSUE TITLE
MODULE_NOT_FOUND for pdf.js

### ISSUE THREAD

**Issue by skepticalwaves** [opendocsg/pdf2md#53]
```
Need to install the following packages:
  @opendocsg/pdf2md
Ok to proceed? (y) y
node:internal/modules/cjs/loader:927
  throw err;
  ^

Error: Cannot find module 'pdfjs-dist/es5/build/pdf'
Require stack:
- /home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/util/pdf.js
- /home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/pdf2md.js
- /home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/pdf2md-cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/util/pdf.js:1:15)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/util/pdf.js',
    '/home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/pdf2md.js',
    '/home/node/.npm/_npx/404aa1a6362bd464/node_modules/@opendocsg/pdf2md/lib/pdf2md-cli.js'
  ]
}

```

**Comment by skepticalwaves** [comment]
Looks like pdfjs got updated in such a way that they broke the API you were using?

**Comment by ptsefton** [comment]
Any progress on this one @skepticalwaves did you get it working?

**Comment by skepticalwaves** [comment]
I have not because I have zero experience in js/node, just a user.

**Comment by ptsefton** [comment]
Is this fixed then? I tried to install it yesterday and had the same problem.

**Comment by LoneRifle** [comment]
Just fixed it and published it in 8c6a7e4, give it a try! You're looking for v0.1.25.

Thanks for your patience!
